src/EventProducerService.cpp

```diff
@@ -213,26 +221,43 @@ void EventProducerService::handleProdSvcMessage(const VlcbMessage *msg)
 {
   unsigned int opc = msg->data[0];
   unsigned int nn = Configuration::getTwoBytes(&msg->data[1]);
-  // DEBUG_SERIAL << ">VLCBProdSvc handling message op=" << _HEX(opc) << " nn=" << nn << " en" << en << endl;
-
-  switch (opc) 
-  {    
-
-    case OPC_AREQ:
-      // AREQ message - request for node state, only producer nodes
-
-      if ((nn == module_config->nodeNum) && (eventhandler != nullptr)) 
+  unsigned int en = Configuration::getTwoBytes(&msg->data[3]);
```

Comment:
Reminder of what we discussed:
* Review what NN is set to for long and short events. See description of ACON/ASON and compare this with AREQ/ASRQ. The op-code spec is not clear enough here. May need to update the op-code spec too.
* Ignore consumer events.

Reply:
All done now and tested.